Makefile

```diff
@@ -43,8 +43,6 @@ deploy: manifests
 	cd config/default && if [ ${KFSERVING_ENABLE_SELF_SIGNED_CA} != false ]; then \
 	kustomize edit remove resource certmanager/certificate.yaml; \
 	else kustomize edit add resource certmanager/certificate.yaml; fi;
-	kubectl get crd inferenceservices.serving.kubeflow.org && kustomize build config/default/crd | kubectl replace --validate=false -f - || \
```

Comment:
The problem currently with `kubectl apply` is that it takes long time(almost 10-15minutes) to install the crd because of the last applied annotation, `kubectl replace` is much faster way.

Reply:
Ah, okay did not know this. I went ahead and put it back.

However, apply commands for the KFServing CRDS have always been pretty quick for me. Is the 10-15 min time just on the testing infra? I found this issue: https://github.com/kubernetes/kubernetes/issues/91615 which outlines what is possibly making it slow, so I downgraded to a kubectl version that didn't have the patch for this issue (v1.18.3) and sure enough, I found myself with a 12 min apply time versus what I had using version v1.18.8 (<5 second apply time). Seems like kubectl should be updated on the testing infra?

Reply:
wow, good finding did not know that it is fixed in newer kubectl version, ye looks like I am using v1.18.3 as well.

Reply:
Since https://github.com/kubeflow/kfserving/pull/1401 was merged, I went ahead and removed the create/replace lines